doc: Add Brands section to documentation (#3552)
Orange-OpenSource/Orange-Boosted-Bootstrap: site/src/content/docs/foundation/themes.mdx

---
title: Themes and brands
description: Documentation and examples for themes usage guidelines.
aliases:
  - "/docs/foundation/themes/"
  - "/docs/about/brand/"
toc: true
---

import { getVersionedDocsPath } from '@libs/path'

> The OUDS Web design system offers three distinct themes: Orange, Orange Compact, and Sosh, representing two brands—Orange and Sosh. Each theme is applied consistently across all elements of the design system, featuring its own color scheme, typography, grid, logo, and more. These themes are intended for use in different contexts and for various purposes, and they should not be mixed within the same project. Each project must select one theme and apply it uniformly across all screens.

Reviewer: ```suggestion
OUDS Web offers three distinct themes: Orange, Orange Compact, and Sosh, representing two brands—Orange and Sosh. Each theme is applied consistently across all elements of the design system, featuring its own color scheme, typography, grid, logo, and more. These themes are intended for use in different contexts and for various purposes, and they should not be mixed within the same project. Each project must select one theme and apply it uniformly across all screens.
```

> This page introduces the three OUDS Web themes. Throughout the rest of the documentation, you can switch themes using the menu in the header.

Reviewer: ```suggestion
This page showcases the three OUDS Web themes. Throughout the rest of the documentation, you can switch themes using the menu in the header.
```
Wondering if we should put the Theme picker icon between `the`and `menu` in this sentence?

Reviewer 2: I don't think we need the icon tbh

> <Callout type="negative" title="Using multiple themes in the same project">

Reviewer: ```suggestion
<Callout type="negative" title="Do not use multiple themes in the same project">
```

>   OUDS Web is designed for use with a single theme per project. However, this page demonstrates multiple themes as a demo. This practice should not be replicated in actual projects, as it can create confusion and inconsistency in design.

Reviewer: ```suggestion
  OUDS Web is designed for use with a single theme per project. This page serves as a themes showcase and should not be replicated in actual projects, as it can create confusion and inconsistency in design.
```

</Callout>

<ThemeDemoStylesImport />

## Orange

The Orange theme represents the default Orange brand for OUDS Web. It is suitable for a wide range of contexts and purposes, featuring Orange’s vibrant color scheme, modern typography, and distinctive logo.

<ThemeDemo theme="orange"/>

### Logo for Orange theme

> OUDS Web provides a single SVG file for both master and small Orange logos, since it’s lightweight and easily made responsive. Here is the unminified SVG content:

Reviewer: ```suggestion
OUDS Web provides a single SVG file for both main and small Orange logos, since it’s lightweight and easily made responsive. Here is the unminified SVG content:
```

Reviewer 2: I think it's a naming used by design. It was already used as `master` in Boosted

<img src={getVersionedDocsPath('assets/brand/orange-logo.svg')} class="figure-img img-fluid m-auto mb-medium" width="50" height="50" alt="Orange logos" loading="lazy" />

<SvgDocs theme="orange" buttonLabel="Orange logo" />

## Orange compact

The Orange Compact theme presents the Orange brand in a more condensed format. It is intended for contexts where a significant amount of information needs to be displayed on each screen, such as business tools or operational interfaces. The Orange Compact theme shares the same color scheme, typography, and logo as the Orange theme.

<ThemeDemo theme="orange-compact" />

### Logo for Orange compact theme

> OUDS Web provides a single SVG file for both master and small Orange logos, since it’s lightweight and easily made responsive. It is the same logos as the ones used in the Orange theme. Here is the unminified SVG content:

Reviewer: ```suggestion
OUDS Web provides a single SVG file for both main and small Orange logos, since it’s lightweight and easily made responsive. It is the same logos as the ones used in the Orange theme. Here is the unminified SVG content:
```
